src/nodes/Network/IPFSGet.js

```diff
@@ -17,52 +14,38 @@ IPFSGet.title = "IPFSDownload";
 
 IPFSGet.prototype.onAdded = async function() {
   this.title_color = "#dddddd";
-  this.ipfs = new IPFS({
+  this.ipfs = await IPFS.create({
     EXPERIMENTAL: {
-     pubsub: true
+     pubsub: true,
    },
-   repo: 'ipfs-' + Math.random(),
    config: {
      Addresses: {
-       Swarm: ['/dns4/ws-star.discovery.libp2p.io/tcp/443/wss/p2p-websocket-star']
-     }
+       Swarm: ['/dns4/wrtc-star1.par.dwebops.pub/tcp/443/wss/p2p-webrtc-star',
+       '/dns4/wrtc-star2.sjc.dwebops.pub/tcp/443/wss/p2p-webrtc-star']
+     },
+     Bootstrap: []
    }
   })
-  await this.ipfs.ready
   console.log('IPFS (get) node is ready')
   const { id, agentVersion, protocolVersion } = await this.ipfs.id()
   console.log("IPFS FOR GET!",id, agentVersion, protocolVersion)
   this.title_color = "#eeee44";
 };
 
 IPFSGet.prototype.onAction = async function() {
-  try{
-    this.result = await this.ipfs.get(this.path)
-    if(this.result && this.result[0]){
-      this.data = this.result[0].content
-      if(typeof this.data.toString == "function"){
-        this.data = this.data.toString()
-      }
-    }
-  }catch(e){console.log(e)}
-}
-
-IPFSGet.prototype.onExecute = async function() {
   let path = this.getInputData(0)
-  if(path && (!this.path || path!=this.path )){
-    this.path = path
-    console.log("PATH!",this.path)
+  if(typeof path !== "undefined" && path != null) {
     try{
-      this.result = await this.ipfs.get(this.path)
-      if(this.result && this.result[0]){
-        this.data = this.result[0].content
-        if(typeof this.data.toString == "function"){
-          this.data = this.data.toString()
+        const results = []
+        for await (const result of this.ipfs.cat(path)) {
+          results.push(Buffer.from(result).toString('utf-8'))
         }
-      }
-    }catch(e){console.log(e)}
-
+        this.data = results[0] 
```

Comment:
converting from a uint8array to a string and accessing using 'cat' instead of 'get' since IPFSAdd is sending a single string argument and file structure requires an object. This will get the content and display the string saved using the IPFSUpload module.